supabase/pending/opengraph_changes_1.sql

```diff
@@ -0,0 +1,85 @@
+-- This script was generated by the Schema Diff utility in pgAdmin 4
```

Comment:
@jgraettinger I talked to @psFried and he suggested I ask for your eyeballs on this. I got it to auto-generate, but had to delete a bunch of grant stuff and change the permissions for `live_specs_ext` to be owned by `authenticated`, not just `grant all`. Anything look wrong to you?